packages/composite/dbaas/exoscale/redis.yml

```diff
@@ -0,0 +1,147 @@
+classes:
+  - '...common'
+parameters:
+  appcat:
+    composites:
+      "exoscaleredis.appcat.vshn.io":
+        spec:
+          group: appcat.vshn.io
+          names:
+            kind: XExoscaleRedis
+            plural: xexoscaleredis
+          claimNames:
+            kind: ExoscaleRedis
+            plural: exoscaleredis
+          connectionSecretKeys:
+            - host
```

Comment:
Can we do all upper case or all lowercase? doesn't seem right to me to have this mixed.

Reply:
but that's how it's exposed in the connection secret.

I'm not sure either if it's that grave, TBH. An application might mount `REDIS_URL` directly as an env variable while another might individually get the other keys in some other way, depending on the use case? 
But no idea what is best here - happy to change but that would require a change in provider-exoscale, too.

Reply:
I would go ahead and update the provider and make it consistent with other configurations like [postgres](https://github.com/vshn/component-appcat/blob/master/packages/composite/dbaas/exoscale/postgres.yml#L16) or [mysql](https://github.com/vshn/component-appcat/blob/master/packages/composite/dbaas/exoscale/mysql.yml#L16).

Reply:
strictly speaking though, the ticket for provider-exoscale mentioned the variables in lower-case, that's why Michael left it according to the ticket.

Reply:
I thought for a second that this might have been done on purpose during the provider-exoscale implementation. @ccremer do you think we should leave as is at this point?

Reply:
IMO we wait for https://github.com/vshn/provider-exoscale/pull/58 and incorporate the changes here.

Reply:
fixed and updated